addon/gql/queries/form-editor-question.graphql

```diff
@@ -1,4 +1,35 @@
 # import * from 'ember-caluma/gql/fragments/question-info'
+# import * from 'ember-caluma/gql/fragments/field-question'
+
+fragment SimpleAnswer on Answer {
```

Comment:
We already have that exact fragment in `gql/fragments/simple-answer.graphql`, why duplicate it?

Reply:
Oversight. I'll use the fragment.

Reply:
Done